### PR TITLE
Favicon helper

### PR DESCRIPTION
**Pull request by michaeldeol**
This pull request attempts to fix issue #400.

### CODE REVIEW

File: lib/lotus/helpers/asset_tag_helpers.rb [hunk 1]
```diff
@@ -103,6 +103,34 @@ def video(src = nil, options = {}, &blk)
 
         html.video(blk, options)
       end
+
+      # Creates a favicon tag with an optional path.
+      #
+      # @since 0.6.0
```
Comment (joneslee85) [inline lib/lotus/helpers/asset_tag_helpers.rb]: s/0.6.0/x.x.x

Comment (michaeldeol) [inline lib/lotus/helpers/asset_tag_helpers.rb]: @joneslee85 Fixed, thanks for catching that.

**Comment by michaeldeol** [comment]
@joneslee85 I thought it was strange that my rebase didn't have more merge conflicts then it did. Looks like the other person touched different parts of the same file. I have removed the duplicate code and travis should now pass.

**Comment by joneslee85** [comment]
@michaeldeol yes, I have merged #408 before you, can you please make another PR with just integration test?

**Pull request merged**